### PR TITLE
Expand imageattr text

### LINKED ISSUE #155: Usage of picture aspect ratio in a=imageattr
B. Section 3.6.1:

   Otherwise, an "a=imageattr" attribute is created with "recv"
   direction, and the resulting resolution space formed by intersecting
   the decoder limits and constraints is used to specify its minimum and
   maximum x= and y= values.  If the intersection is the null set, i.e.,
   there are no resolutions that are permitted by both the decoder and
   the mandatory constraints, this SHOULD be represented by x=0 and y=0
   values.

   The rules here express a single set of preferences, and therefore,
   the "a=imageattr" q= value is not important.  It SHOULD be set to
   1.0.

What about end-points that can produce and consume different aspect ratios on its video. Such a system is likely to have different constraints on its min and max resolution in X and Y for 1:1, 4:3, 16:9 and 1:2.35 aspect ratios. It is unclear if that is allowed, and even recommend practice. And in this case there might be a usage for the q value.

### PR DESCRIPTION
**Pull request by juberti**
Fixes #155, #143
Discusses multiple imageattr; use of different codecs; no aspect ratio
handling; preserve aspect ratio when scaling

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -629,19 +629,32 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             <t>This specification prescribes more specific behavior.
             When a sender of a given MediaStreamTrack, which is producing video
             of a certain resolution, receives an "a=imageattr recv"
-            attribute, it MUST first check to see if the original resolution
-            meets the criteria specified in the attribute, and transmit it
-            untouched if so. Note that when considering a MediaStreamTrack that
+            attribute, it MUST check to see if the original resolution
+            meets the size criteria specified in the attribute, and adapt the
+            resolution accordingly by scaling (if needed).
+            Note that when considering a MediaStreamTrack that
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: Maybe "(if needed and allowed by policy)"

**Comment by ekr** [comment]
LGTM

File: draft-ietf-rtcweb-jsep.xml (hunk 1 above)
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: "(if appropriate)"?

**Comment by fluffy** [comment]
LGTM

**Pull request merged**